include/libtorrent/aux_/session_impl.hpp

```diff
@@ -222,6 +222,7 @@ namespace libtorrent
 			struct session_plugin_wrapper : plugin
 			{
 				explicit session_plugin_wrapper(ext_function_t const& f) : m_f(f) {}
+				explicit session_plugin_wrapper(session_plugin_wrapper const& p) : m_f(p.m_f) {}
```

Comment:
I'm not sure what it means for a copy constructor to be explicit.. also, I think you could just say = default; here, no?